src/test/java/com/game/Battleship/models/BoardTests.java

```diff
@@ -0,0 +1,29 @@
+package com.game.Battleship.models;
+import org.junit.jupiter.api.Test;
+import  static org.junit.jupiter.api.Assertions.*;
+
+public class BoardTests {
+
+    @Test
+    public void addShipsOnTheBoardHorizontal(){
+        Board board = new Board();
+        Ship  cruiser = new Ship("Cruiser",3);
+        board.addShip(cruiser, 1);
+        assertEquals(board.getBoardField().get(1).getValue(), "Cruiser");
+        assertEquals(board.getBoardField().get(2).getValue(), "Cruiser");
+        assertEquals(board.getBoardField().get(3).getValue(), "Cruiser");
+
+    }
+
+    @Test
+    public void addShipsOnTheBoardVertical(){
```

Comment:
What is this new `isHorizontal` attribute and how does setting it to `false` in this test differ from the test above?

Reply:
My thinking is, when I will add a ship to the board(do a patch request) I will sent the `isHorizontal = false` if the ship will be vertical

Reply:
I see. Is the placing ships vertically/horizontally functionality working?

Reply:
yes it does!

Reply:
Hmm, maybe I'm not understanding correctly, but it still seems like this ship is placed horizontally. I'd expect to see something like:

```java
assertEquals(board.getBoardField().get(1).getValue(), "Destroyer");
assertEquals(board.getBoardField().get(11).getValue(), "Destroyer"); // changed from 2
```

If you were to add only this line `cruiser.setIsHorizontal(false)` to the previous test, would you expect that test to fail?

Reply:
Brian, yes you were right. I forgot to delete some extra code and correct the test. I've just did it.Sorry about the confusion

Reply:
Lol yeah I saw that you changed it almost right after I left the comment 😅

Reply:
I've seen the comment after I changed it :)